JS1064_JS104_create_do_while_loop/script.js

```diff
@@ -0,0 +1,8 @@
+// Display the prompt dialogue while the value assigned to `secret` is not equal to "sesame"
+let password = 'abc123';
```

Comment:
remove the initial value
```suggestion
let password = '';
```